VirtoCommerce.CatalogModule.Web.Core/Converters/CategoryConverter.cs

```diff
@@ -39,7 +39,8 @@ public static webModel.Category ToWebModel(this moduleModel.Category category, I
             var parents = new List<moduleModel.Category>();
             if (category.Parents != null)
             {
-                retVal.Outline = string.Join("/", category.Parents.Select(x => x.Id));
+                retVal.Outline = OutlineHelper.EnsurePhysicalOutline(category.Outlines, category.CatalogId)
```

Comment:
This code will add physical catalog ID to every outline, even if it's not physical, and take the first one.
Instead, it should find physical outline and return its string representation.